File: python/7 kyu/88.py
# https://www.codewars.com/kata/5cb05eee03c3ff002153d4ef/train/python

# Passed

def get_section_id(scroll, sizes):
    current_position = 0
    for idx, size in enumerate(sizes):
        current_position += size
        if current_position > scroll:
            return idx

    return -1 # past last section

output = get_section_id(300, [300, 200, 400, 600, 100])
print(output)

File: nodejs/7 kyu/88.js
// https://www.codewars.com/kata/541da001259d9ca85d000688/train/javascript

// Passed

function seqlist(first, c, l) {
  let sequence = [first];
  for (let termNumber = 1; termNumber < l; termNumber++) {
    let term = first + termNumber * c;
    sequence.push(term);
  }

  return sequence;
}

const output = seqlist(0, -5, 8);
console.log(output);